src/moonlink_service/src/test.rs

```diff
@@ -227,6 +291,13 @@ async fn test_schema() {
         response.status().is_success(),
         "Response status is {response:?}"
     );
+    let create_response: serde_json::Value = response.json().await.unwrap();
```

Comment:
I'm wondering if we could get a concrete type directly?
https://github.com/Mooncake-Labs/moonlink/blob/6014832a30dba675f7acbec9404ec34884b08048/src/moonlink_service/src/test.rs#L180